tgui/packages/tgui/interfaces/AccountsUplinkTerminal.js

```diff
@@ -0,0 +1,286 @@
+import { createSearch, decodeHtmlEntities } from 'common/string';
+import { Fragment } from 'inferno';
+import { useBackend, useLocalState } from "../backend";
+import { Box, Button, Flex, Icon, Input, LabeledList, NumberInput, Section, Table, Tabs } from '../components';
+import { FlexItem } from '../components/Flex';
+import { Window } from "../layouts";
+import { ComplexModal, modalOpen } from './common/ComplexModal';
+import { LoginInfo } from './common/LoginInfo';
+import { LoginScreen } from './common/LoginScreen';
+import { TemporaryNotice } from './common/TemporaryNotice';
+
+export const AccountsUplinkTerminal = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    loginState,
+    currentPage,
+  } = data;
+
+  let body;
+  if (!loginState.logged_in) {
+    return (
+      <Window resizable>
+        <Window.Content>
+          <LoginScreen />
+        </Window.Content>
+      </Window>
+    );
+  } else {
+    if (currentPage === 1) {
+      body = <AccountsRecordList />;
+    } else if (currentPage === 2) {
+      body = <DetailedAccountInfo />;
+    } else if (currentPage === 3) {
+      body = <CreateAccount />;
+    }
+  }
+
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <LoginInfo />
+        {body}
+      </Window.Content>
+    </Window>
+  );
+};
+
+const AccountsRecordList = (properties, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    accounts,
+  } = data;
+  const [searchText, setSearchText] = useLocalState(context, "searchText", "");
+  const [sortId, _setSortId] = useLocalState(context, "sortId", "owner_name");
+  const [sortOrder, _setSortOrder] = useLocalState(context, "sortOrder", true);
+  return (
+    <Flex direction="column" height="100%">
+      <AccountsActions />
+      <Section flexGrow="1" mt="0.5rem">
```

Comment:
Avoid using flex shorthands and use the `Flex.Item` component instead.

Reply:
So put it inside the section?

Reply:
The opposite, the Section inside the `Flex.Item`

Reply:
But I want the section to fill the full height for a backdrop

Reply:
Make the `Flex.Item` stretch with `grow="1"` then make the `Section` stretch as well with `height="100%"`

Reply:
Done